cmake/pcl_find_sse.cmake

```diff
@@ -62,7 +62,7 @@ macro(PCL_CHECK_FOR_SSE)
           __m128i vc = _mm_cmpgt_epi64 (va, vb);
 
           _mm_storeu_si128 ((__m128i*)c, vc);
-          if (c[0] == -1LL && c[1] == 0LL)
+          if(c[0] == -1LL && c[1] == 0LL)
```

Comment:
This is not _cmake_ code ;)

Reply:
You are right, I forgot that some files are not pure CMake ;-)